### ISSUE TITLE
Remove recommendation to hack core

### ISSUE THREAD

**Issue by yorkshire-pudding** [backdrop-contrib/ip2country#1]
The readme suggests you may need to modify a function in core in order to allow cron enough time to run. I don't think it is necessary.  I ran a test from a backdrop demo (on b.org) and ARIN updated (including first download) within 10 seconds.  In a local (lando) install in UK, it took 17 seconds.

At the very least a recommendation like this should not be in the README but possible as part of a troubleshooting in a wiki if it really needs to be there.

If there is a genuine case that certain modules need longer cron times, do we need a change to core to allow modules to request this?

**Comment by docwilmot** [comment]
This is true. Perhaps this is a relic of times when internet was globally dial-up and slow. It should not be necessary for the majority now I imagine. Cn you PR please?
